### ISSUE TITLE
Link to create issue on every webpage

### ISSUE THREAD

**Issue by kwk** [almighty/almighty-ui#66]
What do you think about having a link in the UI that you can click at any time to create a github issue with pre-filled fields?

For instance, this link creates an issue pre-filled with some values like commit id, page URL.

https://github.com/almighty/almighty-ui/issues/new?title=%3CFILL_IN_PAGE_TITLE%3E&body=%23+Details%0A%20*%20Commit%20ID:%20COMMIT_ID%0A%20*%20URL%20ID:%20URL

Renders as:

# Details
 * Commit ID: COMMIT_ID
 * URL ID: URL

**Comment by kwk** [comment]
This link might help in building a proper URL with markdown in the body: http://www.w3schools.com/tags/ref_urlencode.asp

**Comment by joshuawilson** [comment]
@kwk is this a link in the actual UI for ALMighty or in the README?

@aslakknutsen have you seen other projects do this sort of thing?

**Comment by aslakknutsen** [comment]
@joshuawilson Yeah, it's essentially a 'Report an Issue' feature that can capture some context. Would be nice if we could have screenshots etc as well, but.. 

If not in production, it makes sense for a dev/stage build

**Comment by joshuawilson** [comment]
I like the idea.

**Comment by mindreeper2420** [comment]
Is this in the ALMighty UI or Github repo? I like the idea for in the ALMighty UI and would be up for creating mockups and flows for this.

**Comment by aslakknutsen** [comment]
@mindreeper2420 almighty ui

**Comment by kwk** [comment]
One could also include the request and the response from UI to core for technical analysis (if there's any).

**Comment by michaelkleinhenz** [comment]
I think we need that also for automatic issue creating by other systems, like BAMs or similar reactive stuff. There should be an "webhooks API" for creating all sorts of issues and links. Example usecase: a customer ticket system detects a technical problem. It should be able to create a ticket AND link the new ticket to his own ticket(s). Usually the integration in those systems will be "no code access, use some webhooks or very simple plugin system".
